DevelopGitHub/GitHubExtension/GitHubExtension.Tools.LocalizationTool/Model/TranslationDataRow.cs

```diff
@@ -0,0 +1,94 @@
+﻿using GitHubExtension.LocalizationTool.Translate;
+using GitHubExtension.LocalizationTool.ViewModel;
+
+namespace GitHubExtension.LocalizationTool.Model
+{
+    public class TranslationDataRow : NotifyProperyChangedBase
+    {
+        private string _name;
+
+        private string _ru;
+
+        private string _uk;
+
+        private string _en;
+
+        public TranslationDataRow()
+        {
+        }
+
+        public TranslationDataRow(string name)
+        {
+            Name = name;
+        }
+
+        public string Name
+        {
+            get { return _name; }
+            set { CheckPropertyChanged(ref _name, value); }
```

Comment:
Because I add method that take lambda expression and then get caller member name from this expression. You asked me "Why we need this old code?". So I thought we decided not to use lambdas. Now I'm using CallerMemberName atribute because it works faster than lambdas.